test/services/sync/strategies/CSVSyncer/CSVSyncer.unit.test.js

```diff
@@ -75,5 +78,23 @@ describe('CSVSyncer', () => {
 			expect(result[1].class).to.equal('');
 			expect(result[2].class).to.equal('2b');
 		});
+
+		it('should not accept unvalid dates as user birthday', () => {
+			const falsyValues = [false, null, undefined, '', 0]
+			const misfitValues = [Symbol(), [], {}, true]
+			const wrongFormatDates = ['32.12.2000', '01.13.2000', '01.01-2000', '01/01.2000', '01-01/2000', '42', 'void']
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This line has a length of 121. Maximum allowed is 120.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5220139)